cli/cmd/cluster.go

```diff
@@ -753,8 +764,8 @@ func CreateLogGroupIfNotFound(awsClient *aws.Client, logGroup string) error {
 	return nil
 }
 
-// createDashboard creates a new dashboard (or clears an existing one if it already exists)
-func createDashboard(awsClient *aws.Client, dashboardName string) error {
+// createOrClearDashboard creates a new dashboard (or clears an existing one if it already exists)
+func createOrClearDashboard(awsClient *aws.Client, dashboardName string) error {
```

Comment:
maybe change function name because it clears AND creates the dashboard

Reply:
My thinking was that it creates it if it doesn't exist, or it clears it if it did exist. Do you think that makes sense, or you still suggest changing the name?

Reply:
with dashboard clear makes sense